### DISCUSSION TITLE
[ Content Patcher ] And [ Content Patcher Animations ]

### DISCUSSION THREAD

**Discussion by StrangePanda0** [community/community#36743] [Other Feature Feedback, Questions, & Ideas]
can you help me to get yesterday's weather? in numerical, cuze i tried Query Token but it didn't work even directly in Changes > When or DynamicToken > When and DynamicToken > Value, well if i use Random, it'll work but not based on yesterday's weather, can somebody help me?  here's my code.

{
  "Format": "1.19.0",
  "ConfigSchema": {
    "1.5": {
      "AllowValues": "true, false",
      "Default": "false"
    }
  },
  "DynamicTokens": [
    {
      "Name": "Date_Of_Storm",
      "Value": "{{query: {{Day}} + 1}}",
      "When": {
        "Weather": "Storm",
        "Day": "{{Range: 1, 27}}"
      }
    },
    {
      "Name": "Date_Of_Storm",
      "Value": "1",
      "When": {
        "Weather": "Storm",
        "Day": "28"
      }
    },
    {
      "Name": "Weathers",
      "Value": "New_Sun_Wet",
      "When": {
        "Weather": "Sun",
        "Day": "{{Date_Of_Storm}}"
      }
    },
    {
      "Name": "Weathers",
      "Value": "New_Wind_Wet",
      "When": {
        "Weather": "Wind",
        "Day": "{{Date_Of_Storm}}"
      }
    }
  ],
  "Changes": [
    {
      "LogName": "Animation 0001",
      "Action": "EditImage",
      "Target": "Maps/{{Season}}_outdoorsTileSheet",
      "FromFile": "Assets/Environment/{{Weathers}}/{{Season}}_01.png",
      "FromArea": {
        "X": 0,
        "Y": 0,
        "Width": 16,
        "Height": 176
      },
      "ToArea": {
        "X": 0,
        "Y": 96,
        "Width": 16,
        "Height": 176
      },
      "AnimationFrameCount": 16,
      "AnimationFrameTime": 10
    }
  ]
}

**Comment by ossanna16** [comment]
Hi there @StrangePanda0,
The GitHub Community on Discussions is a space for all GitHub users to come and discuss ideas and projects. Your post was flagged and removed because it violated our [Community Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md), we ask that members contribute in a positive and constructive way. Posting screenshots of test questions without context will likely discourage engagement and responses.

If this content is posted again, your Community account will be subject to immediate and indefinite suspension.

For additional guidance, please review our [Community Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md) and [GitHub’s Terms of Service](https://help.github.com/articles/github-terms-of-service/).

Thank you!
